__tests__/lib/validations/sql.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import path from 'path';
+import debugLib from 'debug';
+import fetch, { Response } from 'node-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { validate } from 'lib/validations/sql';
+
+const debug = debugLib( '@automattic/vip:__tests__:validations:sql' );
+
+// Mock console.log()
+let output;
+global.console = {
+	log: ( m, d = '' ) => output += m + d + '\n',
+	error: ( m, d = '' ) => output += m + d + '\n',
+};
+jest.spyOn( global.console, 'log' );
+
+const mockExit = jest.spyOn( process, 'exit' ).mockImplementation( () => {} );
+const ERROR_CODE = 1;
+
+jest.mock( 'node-fetch' );
+fetch.mockReturnValue( Promise.resolve( new Response( 'ok' ) ) );
+
+describe( 'lib/validations/sql', () => {
+	describe( 'it fails when the SQL has', () => {
+		beforeAll( async () => {
+			try {
+				output = '';
+				const badSqlDumpPath = path.join( process.cwd(), '__fixtures__', 'validations', 'bad-sql-dump.sql' );
+				await validate( badSqlDumpPath );
+			} catch ( e ) {
+				debug( 'Error:', e.toString() );
+			}
+			debug( 'output', output );
+		} );
+		it( 'an error, and exits with code 1', () => {
+			debug( 'b1' );
+			expect( mockExit ).toHaveBeenCalledWith( ERROR_CODE );
+		} );
+		it( 'USE statements', () => {
```

Comment:
Looks like this is failing the lint check since pulling in the upstream linter changes https://github.com/Automattic/vip/pull/579

Should just need to add a lower-cased word to start these strings.